From c5c3b6e72755dae8b54013f97dfd3f6479c163b1 Mon Sep 17 00:00:00 2001
From: rleix <rayx.lei@intel.com>
Date: Mon, 26 Mar 2018 10:47:10 +0800
Subject: [PATCH] [Email] Fix the loading drawable crash when the display size
 is small

The best resource should be drawable-v21 when the display size is small.
But the file "./drawable-v21/ic_next_24dp.xml" is a vectorgraph and it
is not matched which required bitmap in file "./drawable/ic_next.xml"

Rename it and the vectorgraph can be used directly when the display
size is small.

Test:
1. Goto Settings->Display->Display size, set to "small".
Or using command: adb shell wm density 136
2. Launch email by command:
adb shell am start com.android.email/.activity.setup.AccountSetupFinal

Change-Id: I8c61abd4a5b3d46f9701934d2dac15ba07243ad7
Tracked-On:https://jira01.devtools.intel.com/browse/OAM-66065
Signed-off-by: Lei,RayX <rayx.lei@intel.com>
---
 res/drawable-v21/ic_next.xml      | 24 ++++++++++++++++++++++++
 res/drawable-v21/ic_next_24dp.xml | 24 ------------------------
 2 files changed, 24 insertions(+), 24 deletions(-)
 create mode 100644 res/drawable-v21/ic_next.xml
 delete mode 100644 res/drawable-v21/ic_next_24dp.xml

diff --git a/res/drawable-v21/ic_next.xml b/res/drawable-v21/ic_next.xml
new file mode 100644
index 0000000..91577e1
--- /dev/null
+++ b/res/drawable-v21/ic_next.xml
@@ -0,0 +1,24 @@
+<!--
+Copyright (C) 2014 The Android Open Source Project
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24.0dp"
+        android:height="24.0dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#8a000000"
+        android:pathData="M8.6,16.3l4.599999,-4.499999 -4.599999,-4.6 1.4,-1.4 6.0,6.0 -6.0,5.999999z"/>
+</vector>
diff --git a/res/drawable-v21/ic_next_24dp.xml b/res/drawable-v21/ic_next_24dp.xml
deleted file mode 100644
index 91577e1..0000000
--- a/res/drawable-v21/ic_next_24dp.xml
+++ /dev/null
@@ -1,24 +0,0 @@
-<!--
-Copyright (C) 2014 The Android Open Source Project
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-         http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
--->
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="24.0dp"
-        android:height="24.0dp"
-        android:viewportWidth="24.0"
-        android:viewportHeight="24.0">
-    <path
-        android:fillColor="#8a000000"
-        android:pathData="M8.6,16.3l4.599999,-4.499999 -4.599999,-4.6 1.4,-1.4 6.0,6.0 -6.0,5.999999z"/>
-</vector>
-- 
1.9.1

